config.m4

```diff
@@ -390,7 +390,8 @@ if test "$PHP_MONGODB" != "no"; then
 
     dnl If compiling without libmongocrypt, use kms_message sources bundled with libmongoc.
     dnl If compiling with libmongocrypt, kms_message bundled with libmongocrypt is used as it is most likely newer.
-    if test "$PHP_MONGODB_CLIENT_SIDE_ENCRYPTION" != "yes" && "$PHP_MONGODB_SSL" != "no"; then
+    if test "$PHP_MONGODB_CLIENT_SIDE_ENCRYPTION" != "yes" && test "$PHP_MONGODB_SSL" != "no"; then
+      AC_SUBST(MONGOC_ENABLE_MONGODB_AWS_AUTH, 1)
```

Comment:
Noting that `MONGOC_ENABLE_MONGODB_AWS_AUTH` earlier defaults to `0`, so these are the two instances where we'd actually enable it (i.e. both KMS and TLS are available).